dbft.go

```diff
@@ -123,6 +123,14 @@ func (d *DBFT) InitializeConsensus(view byte) {
 	} else {
 		timeout = d.SecondsPerBlock << (d.ViewNumber + 1)
 	}
+	if d.lastBlockIndex+1 == d.BlockIndex {
+		var ts = d.Timer.Now()
```

Comment:
`var x =` -> `x :=` ?

Reply:
¡No me gusta `:=`! Not that it's that big of a problem, though.